descope/management/access_key.py

```diff
@@ -128,13 +132,14 @@ def update(
         Args:
         id (str): The id of the access key to update.
         name (str): The updated access key name.
+        description(str): an optional text the access key can hold.
```

Comment:
I would make a change here. Either we give the argument a default value `description: str = ""` or change the text so the word "optional" is not there since the argument will be required.
In order to keep the API without compulsory changes, I'd give a default value

Reply:
I would change to : `description(str): The description of the access key to update`, like in the id. 
So it will be consistence.